src/bots/ama/subCommands/editEvent.js

```diff
@@ -0,0 +1,89 @@
+import EventModel from '../models/Event';
+import client from '../../../client';
+import { escapedBackticks } from '../../../utils/index';
+import parseDictionary from '../parser';
+import {
+  getFormattedEvent,
+  getStrings,
+  possibleEditFields
+} from '../constants';
+
+/**
+ * Handles editing an event with Event schema and sends message with updated event
+ * @param {Array.<string>} args - rest of command arguments
+ */
+const handler = async (args) => {
+  if (args.length < 1) {
+    client.message.channel.send(
+      getStrings(possibleEditFields).editEventExample
+    );
+    return;
+  }
+
+  let targetEvent;
+
+  try {
+    targetEvent = await EventModel.findById(args[0]);
+  } catch (_) {
+    client.logger.debug('invalid ObjectId type supplied');
+  }
+
+  if (targetEvent) {
+    if (client.message.content.indexOf(escapedBackticks) === -1) {
+      client.message.channel.send(getStrings().noBackticks);
```

Comment:
is there a reason you're requiring backticks? from what I'm seeing it doesn't necessarily make it easier to parse or anything.

I think we should have it like salary where they aren't required (?)